src/parsers/native_format_parsers.js

```diff
@@ -1,26 +1,19 @@
+import { STRUCTURAL_INFORMATION_FORMATS } from "./enums";
```

Comment:
this file really contains the Factory - should be

```JS
const parser = ParserFactor.getParserByDetectingTextFormat(text);
return parser.parse(text);
```

Reply:
If we want to implement this now in here, we need to touch and change calls to JSONParser, PoscarParser etc and create new format detection function